spec/yaml/swiftnav/sbp/flash.yaml

```diff
@@ -64,15 +64,51 @@ definitions:
           size_fn: addr_len
           desc: Data to program addresses with, with length N=addr_len
 
+ - MSG_FLASH_PROGRAM_OLD:
+    id: 0x00E0
+    public: False
+    short_desc: Deprecated, use MSG_FLASH_PROGRAM (0x00E6)
+    desc: |
+      Deprecated, use MSG_FLASH_PROGRAM (0x00E6).
+    fields:
+      - target:
+          type: u8
+          desc: Target flags
+          fields:
+            - 1-7:
+                desc: Reserved
+            - 0:
+                desc: Flash target to read
+                values:
+                  - 0: FLASH_STM
+                  - 1: FLASH_M25
+      - addr_start:
+          type: array
+          fill: u8
+          size: 3
+          units: bytes
+          desc: Starting address offset to program
+      - addr_len:
+          type: u8
+          units: bytes
+          desc: |
+            Length of set of addresses to program, counting up from
+            starting address
+      - data:
+          type: array
+          fill: u8
+          size_fn: addr_len
+          desc: Data to program addresses with, with length N=addr_len
+
  - MSG_FLASH_DONE:
```

Comment:
Should this be renamed to `MSG_FLASH_DONE` or `MSG_FLASH_DEVICE` or something to be consistent with the naming convention?